### ISSUE TITLE
how is the accuracy and efficient of the model?

### ISSUE THREAD

**Issue by runtoforever** [tefantasy/CorrNet#2]
Hello, I want to know your experimental accuracy of running based on this code, Is it high, low or about the same? and  how efficient is the network?

**Comment by tefantasy** [comment]
Hi, the results of our experiments have been updated in README.md. 

For efficiency:
We use 32x Tesla V100 32GB GPUs. With overall batch_size=256 (8 per GPU, 64 per node), the memory footprint is ~16GB for each GPU. A complete training procedure takes ~12 days, because our implementation of the correlation operator is not very time efficient (we use the custom CUDA implementation instead of pytorch API for correlation operation in these experiments).

**Comment by runtoforever** [comment]
Thanks for the update, your experimental results on Kinetics600 are higher than the paper. Do you still keep the trained model and Can you send me a copy?

**Comment by tefantasy** [comment]
I am sorry that I cannot share the trained weights of the model.

**Comment by runtoforever** [comment]
Hello, I am mainly used for postgraduate research, and there is no other purpose. In the case of no pre-training model, the loss always is nan so that graduate projects can't go on. Please!!!

**Comment by tefantasy** [comment]
Again I am not able to provide you the trained model.

For the nan loss issue, you can try to:
1) decrease your learning rate.
2) apply gradient clipping to avoid gradient explosion. 

If you are training on your own dataset, you could try to find some pretrained weights of R(2+1)D model in other repositories, and initialize your network except the correlation blocks with those weights.  This should give a better performance than randomly initialize the whole network.

**Comment by tefantasy** [comment]
If you have further questions, please open a new issue.
